copilot-instructions.md

```diff
@@ -0,0 +1,408 @@
+# GitHub Copilot Instructions for Sitecore Content SDK
+
+## Project Overview
+
+This is the **Sitecore Content SDK** - a comprehensive TypeScript/JavaScript SDK for building modern web applications with Sitecore XM Cloud. The project provides core functionality, React components, Next.js integration, and CLI tools for scaffolding applications.
+
+### Tech Stack
+- **Language**: TypeScript (Node LTS)
+- **Runtime**: Node LTS or higher
+- **Build**: `tsc` -> `dist/`, templates bundled via `scripts/build-templates.ts`
+- **Testing**: Mocha + Sinon + Chai, coverage via `nyc`
+- **Lint/format**: ESLint + Prettier
+- **Package Manager**: Yarn 3.1.0
+
+### Core Packages
+- `@sitecore-content-sdk/core` - Core SDK functionality
+- `@sitecore-content-sdk/react` - React components and hooks
+- `@sitecore-content-sdk/nextjs` - Next.js integration and middleware
+- `@sitecore-content-sdk/cli` - CLI tools and scaffolding
+- `@sitecore-content-sdk/create-content-sdk-app` - Scaffolding CLI & templates
```

Comment:
Both CLI's are marked as "scaffolding". May be "create-content-sdk-app" can be rephrased a bit to be clear that it's used to scaffold templates?
```suggestion
- `@sitecore-content-sdk/cli` - CLI tools and scaffolding
- `@sitecore-content-sdk/create-content-sdk-app` - CLI for template scaffolding
```